cmd/aries-agent-mobile/pkg/wrappers/rest/rest.go

```diff
@@ -98,8 +92,24 @@ func makeHTTPRequest(httpClient httpClient, method, agentURL, token string, body
 	return ioutil.ReadAll(response.Body)
 }
 
-func embedPIID(reqPath string, body []byte) (string, error) {
-	if body == nil || !strings.Contains(reqPath, "{piid}") {
+func embedParams(reqPath string, body []byte) (newURL string, err error) {
+	params := []string{"piid", "id", "name"}
+	newURL = reqPath
+
+	for _, param := range params {
+		newURL, err = embedParam(newURL, param, body)
+		if err != nil {
+			return
```

Comment:
Not catching error here ?

Reply:
the return types are named on line https://github.com/hyperledger/aries-framework-go/pull/2012/files#diff-a6ca25429bc87f0f1c99523c587d7d4cR95